<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmlDesigner::DebugViewWidget</class>
 <widget class="QWidget" name="QmlDesigner::DebugViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debug</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="instanceLog_2">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Model Log</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="1" column="0">
        <widget class="QPlainTextEdit" name="modelLog">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="modelClear">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>337</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Instance Notifications</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="1" column="0">
        <widget class="QPlainTextEdit" name="instanceLog">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="instanceLogClear">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Instance Errors</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="1" column="0">
        <widget class="QPlainTextEdit" name="instanceErrorLog">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="instanceErrorClear">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>337</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="enabledCheckBox">
     <property name="text">
      <string>Enabled</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>modelClear</sender>
   <signal>clicked()</signal>
   <receiver>modelLog</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>instanceLogClear</sender>
   <signal>clicked()</signal>
   <receiver>instanceLog</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>instanceErrorClear</sender>
   <signal>clicked()</signal>
   <receiver>instanceErrorLog</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
